src/quo2/components/navigation/page_nav.cljs

```diff
@@ -31,17 +31,17 @@
            {:no-color true})))
 
 (defn left-section-view
-  [{:keys [on-press icon accessibility-label type icon-background-color]
+  [{:keys [on-press icon accessibility-label type icon-background]
     :or   {type :grey}}
    put-middle-section-on-left?]
   [rn/view {:style (when put-middle-section-on-left? {:margin-right 5})}
    [button/button
-    {:on-press                  on-press
-     :icon                      true
-     :type                      type
-     :size                      32
-     :accessibility-label       accessibility-label
-     :override-background-color icon-background-color}
+    {:on-press            on-press
+     :icon-only?          true
+     :type                type
+     :size                32
+     :accessibility-label accessibility-label
+     :background          icon-background}
```

Comment:
there is a better solution for this, however this component needs to be revisted as it can be much simpler if we encapsulate the colors/designs correctly in the component 👍